nupic/research/frameworks/pytorch/model_utils.py

```diff
@@ -310,3 +330,30 @@ def deserialize_state_dict(fileobj, device=None):
         # FIXME: Backward compatibility with old uncompressed checkpoints
         state_dict = torch.load(fileobj, map_location=device)
     return state_dict
+
+
+def freeze_output_layer(model, indices, layer_type="dense", linear_number=2):
+    """ Freeze output layer gradients of specific classes for classification.
```

Comment:
Looks like this function is still in the main model_utils.py in the PR? You should take it out now that you've implemented it elsewhere.